File: prime.c
#include <stdio.h>

int prime(int x)
{
        int f = 1;
        for (int i = 2; i < x / 2 + 1; i++) {
                if (x % i == 0) { f = 0; break; }
        }
        return (f);
}

int main(void)
{
        int x, n;
        printf("x = ? "); scanf("%d", &x);
        for (n = 2; n <= x; n++) if (prime(n)) printf("%d ", n);
        printf("\n");
        return (0);
}

File: prime.py
def prime(x):
    i = 2
    while i < x / 2 + 1:
        if x % i == 0:
            return(False)
        i = i + 1
    return(True)

x = int(input("x = ? "))
for i in range(1,x):
    if prime(i+1):
        print(i+1, end=" ")
print()
